components/places/src/ffi.rs

```diff
@@ -131,6 +132,13 @@ impl PlacesApi {
         )?;
         Ok(serde_json::to_string(&ping).unwrap())
     }
+
+    // TODO: Relook into this and fix appropiately
```

Comment:
as above, I think just removing this from this PR is fine. We probably do need a new ticket for "remove all old handle cruft and ensure we have appropriate cleanup/close functions for the new world"